package.json

```diff
@@ -13,12 +13,26 @@
     "block"
   ],
   "repository": "ybiquitous/remark-lint-code-block-syntax",
-  "main": "index.js",
-  "files": [],
+  "engines": {
+    "node": ">=12"
+  },
+  "main": "./index.cjs",
+  "exports": {
+    "import": "./index.js",
+    "require": "./index.cjs"
+  },
+  "type": "module",
+  "files": [
+    "index.js",
+    "index.cjs"
+  ],
   "scripts": {
-    "test": "jest",
-    "lint": "npx eslint . && npx prettier --check .",
-    "format": "npx prettier --write ."
+    "build": "npx rollup index.js --file=index.cjs --format=cjs --exports=default",
+    "pretest": "npm run build",
+    "test": "NODE_OPTIONS=--experimental-vm-modules jest",
```

Comment:
[note] `NODE_OPTIONS=--experimental-vm-modules` is needed to run Jest:

See <https://jestjs.io/docs/ecmascript-modules>